carbonmark-api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@klimadao/carbonmark-api",
-  "version": "2.0.3",
+  "version": "4.0.0",
```

Comment:
This might be the wrong version bump depending on which PR is merged first.. If you intend to wait on this #1984 then this is ok.